vm/hash_test.go

```diff
@@ -907,49 +946,49 @@ func TestHashMapValuesMethod(t *testing.T) {
 		{`
 		h = { a: 1, b: 2, c: 3 }
 		result = h.map_values do |v|
-		  v * 3
```

Comment:
@saveriomiroddi I prefer keeping the two spaces indentation for those Goby code in test cases